govuk_personalisation.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.description   = "A gem to hold shared code which other GOV.UK apps will use to implement accounts-related functionality."
   spec.homepage      = "https://github.com/alphagov/govuk_personalisation"
   spec.license       = "MIT"
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.6.6")
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.7.5")
```

Comment:
I don't think we're necessarily breaking compatibility with ruby 2.6 here, so I don't think this needs increasing.

Reply:
Ah, this might be needed because Rubocop expects the version here to match the one in `.ruby-version`:

```
Running RuboCop...
RuboCop failed!
Inspecting 26 files
..C.......................

Offenses:

govuk_personalisation.gemspec:17:32: C: Gemspec/RequiredRubyVersion: required_ruby_version and TargetRubyVersion (2.7, which may be specified in .rubocop.yml) should be equal.
  spec.required_ruby_version = Gem::Requirement.new(">= 2.6.6")
```